web/app/scripts/src/directives/form.es.js

```diff
@@ -43,19 +45,18 @@ module.directive('surveysForm', [
                 $scope.$watch('responseUrl', (url) => {
                     if (url) {
                         API.get(url).then(function (data) {
-                            $scope.model = FieldsetParser.parseResponseToModel(data);
+                            $scope.model = data;
                         });
                     }
                 });
 
                 $scope.submit = () => {
```

Comment:
We decided to not define functions this way, as in this should be `= function () {`